sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -2128,10 +2129,14 @@ def from_dict(cls, data):
 class DocumentContentElement(object):
     """A DocumentContentElement.
 
-    :ivar content: Text content of the word.
+    :ivar content: Text content of the document content element.
     :vartype content: str
-    :ivar bounding_box: Bounding box of the word.
+    :ivar bounding_box: Bounding box of the document content element.
     :vartype bounding_box: list[Point]
+    :ivar span: Location of the element in the full document content.
+    :vartype span: ~azure.ai.formrecognizer.DocumentSpan
+    :ivar confidence: Confidence of accurately extracting the document content element.
+    :vartype confidence: float
     :ivar str kind: The kind of document element. Possible kinds are "word" or "selectionMark" which
         correspond to a :class:`~azure.ai.formrecognizer.DocumentWord` or
         :class:`~azure.ai.formrecognizer.DocumentSelectionMark`, respectively.
```

Comment:
Our return statement for the get children methods doesnt have anything to do with the DocumentContentElements right now, so I think we're fine leaving it as is since both classes inherit from and still are content elements. With the individual get words, get lines we arent relating anything to this model.

Reply:
good point!